subprojects/core/src/main/java/org/gradle/api/internal/plugins/DefaultConvention.java

```diff
@@ -164,6 +166,16 @@ public ExtraPropertiesExtension getExtraProperties() {
 
     @Override
     public Map<String, TypeOf<?>> getSchema() {
+        DeprecationLogger.nagUserOfReplacedProperty("ExtensionContainer.schema", "ExtensionContainer.extensionsSchema");
```

Comment:
:x: I don't think this is necessary, it is a recent `@Incubating` API that's most likely only being used by `kotlin-dsl`, the `@Deprecated` annotation for one release cycle should be enough, right?

Reply:
It was introduced as a public incubating API in 3.5. I agree it's most likely only being used by `kotlin-dsl` and the nagging forces us to change the integration tests assertions that will get in our way when integrating a subsequent version of `kotlin-dsl` that don't use the deprecated method anymore. In this context not nagging makes sense to me.

Done, @bamboo could you have another look?